packages/x-charts/src/ChartsSurface.tsx

```diff
@@ -35,7 +35,10 @@ export interface ChartsSurfaceProps {
 const ChartChartsSurfaceStyles = styled('svg', {
   name: 'MuiChartsSurface',
   slot: 'Root',
-})(() => ({}));
+})(() => ({
+  // This prevents default touch actions when using the svg on mobile devices.
```

Comment:
I assume it's the main event you want to disable. Or there are other I ignore?
```suggestion
  // This prevents default touch actions when using the svg on mobile devices.
  // For example, prevent page scroll & zoom.
```

Reply:
Those are mostly it, the api calls "scroll" as a lot of different parts, "pan-horizontal|pan-left|pan-right..."